route_finder/server/src/server/add_explanation.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+from typing import Any
+
+import google.generativeai as genai
+
+
+# configure google api
+# TODO: set the API key when executing the script.
+genai.configure(api_key=os.getenv('GOOGLE_API_KEY', ''))
+model = genai.GenerativeModel('gemini-pro')
+
+
+def add_explanation(additional_info: str, data_geojson_str: str) -> str:
+    # generate prompt
+    prompt: str = (
+        "次のJSONデータを読み込んで、そのデータが示すエリア毎の場所と特徴を説明する文章を生成してください。\n"
+        "説明の内容は下記の内容を考慮してください。\n"
+        f"・{additional_info}の観点を重視してください。"
+        "・エリア毎に、そのエリアの特徴を元にタイトルを付けてください。\n"
+        "・エリア毎に、そのエリアの見どころとその見どころの特徴を3つ挙げてください。\n"
+        "説明の出力フォーマットは下記の通りです。\n"
+        "・エリアのタイトル\n"
+        "・そのエリア全体の特徴\n"
+        "・そのエリアの見どころ\n"
+    )
```

Comment:
nit: I guess using here doc is easier to read and modify?

```suggestion
import textwrap
    prompt: str = textwrap.dedent(f"""
        次のJSONデータを読み込んで、そのデータが示すエリア毎の場所と特徴を説明する文章を生成してください。
        説明の内容は下記の内容を考慮してください。
        ・{additional_info}の観点を重視してください。
        ・エリア毎に、そのエリアの特徴を元にタイトルを付けてください。
        ・エリア毎に、そのエリアの見どころとその見どころの特徴を3つ挙げてください。
        説明の出力フォーマットは下記の通りです。
        ・エリアのタイトル
        ・そのエリア全体の特徴
        ・そのエリアの見どころ
        """.lstrip())
```